.changeset/fresh-shrimps-rhyme.md

```diff
@@ -0,0 +1,12 @@
+---
+"@logto/experience": minor
+---
+
+migrate experience app using Experience API.
+
+Migrate experience app API requests from legacy [Interaction API](https://openapi.logto.io/group/endpoint-interaction) to the new [Experience API](https://openapi.logto.io/group/endpoint-experience), except the following endpoints:
+
+- `GET /api/interaction/consent`
+- `POST /api/interaction/consent`
+
+Those endpoints are used in the third-party application's consent page only. Remain unchanged.
```

Comment:
can we rename to `api/experience/consent`?

Reply:
Have an issue tracking this, will migrate these two APIs under the experience routes.

Reply:
can you paste the issue number here?